Tests/DatadogTests/Datadog/Core/FeaturesConfigurationTests.swift

```diff
@@ -251,6 +251,23 @@ class FeaturesConfigurationTests: XCTestCase {
         )
     }
 
+    func testCustomProxy() throws {
+        let proxyConfiguration: [AnyHashable: Any] = [
+            kCFNetworkProxiesHTTPEnable as AnyHashable: true,
+            kCFNetworkProxiesHTTPPort as AnyHashable: 123,
+            kCFNetworkProxiesHTTPProxy as AnyHashable: "www.example.com",
+            kCFProxyUsernameKey as AnyHashable: "proxyuser",
+            kCFProxyPasswordKey as AnyHashable: "proxypass",
+        ]
```

Comment:
All casting `as AnyHashable` are redundant, it is enforced by the dictionary type.
It also valid for all getter in further assertions.

```suggestion
        let proxyConfiguration: [AnyHashable: Any] = [
            kCFNetworkProxiesHTTPEnable: true,
            kCFNetworkProxiesHTTPPort: 123,
            kCFNetworkProxiesHTTPProxy: "www.example.com",
            kCFProxyUsernameKey: "proxyuser",
            kCFProxyPasswordKey: "proxypass",
        ]
```